umpleonline/umple.php

```diff
@@ -573,4 +572,7 @@
       ); //
   </script>
 </body>
+	<div class="visitors-count" align="right">

```

Comment:
Although this seems to work, any div should be inside the body tag, so the end body tag just above needs to be below the end of this div